app/views/runs/_split_table.slim

```diff
@@ -14,6 +14,7 @@
           - else
             th.align-right Duration
             th.align-right Finished At
+            th.align-right History
```

Comment:
This doesn't seem to be showing up in the table for me, the spot where it goes is just blank.

Reply:
Huh. That's weird, it shows for me. Do the "View" links in the `td`s show up? What browser / OS?

Reply:
![chrome_2017-10-12_21-49-22](https://user-images.githubusercontent.com/7030913/31530735-44503b14-af97-11e7-949c-305a8c4260c4.png)

Both Windows 7 Chrome and OSX Chrome.

Reply:
Amazingly I managed to click gametime runs on 2 different computers to see it happen the 2nd time tonight.